Doc/library/typing.rst

```diff
@@ -164,13 +161,21 @@ compatibility, type aliases can also be created through simple assignment::
 
    Vector = list[float]
 
-Or marked with :data:`TypeAlias` to make it explicit that this is a type alias,
-not a normal variable assignment::
+The following example is an expansion of the above that shows what is
```

Comment:
I don't agree with this sentence and I don't think it clarifies the situation for new users.